app/scripts/loadout/dimLoadoutPopup.directive.js

```diff
@@ -120,7 +119,9 @@
         name: name,
         items: _(items)
           .chain()
-          .select('equipped')
+          .select(function(item) {
```

Comment:
Oh, right, because I used the same function for both. I'll fix it.

On Monday, May 9, 2016, kyle shay <notifications@github.com> wrote:

> In app/scripts/loadout/dimLoadoutPopup.directive.js
> <https://github.com/DestinyItemManager/DIM/pull/507#discussion_r62507739>:
>
> > @@ -120,7 +119,9 @@
> >          name: name,
> >          items: _(items)
> >            .chain()
> > -          .select('equipped')
> > +          .select(function(item) {
>
> we lost .select('equipped') here (per #507 (comment)
> <https://github.com/DestinyItemManager/DIM/pull/507#issuecomment-217879350>
> )
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/DestinyItemManager/DIM/pull/507/files/59e6025dbd26b42a178646f27f558e7f0d8460a2#r62507739>
>